en/admin/control.rst

```diff
@@ -44,19 +44,7 @@ One of the following can be specified in <command>:
 *   status: check status of a database server process. 
 
 | Every command can specify a database name (**[database_name]**) as an argument.
-| If the database name is not specified, the **status** command displays the currently running database servers' information, and the commands except **status** refer to the database names in the **server** property of the **[service]** section of **cubrid.conf**.
-
-::
-
-    # cubrid.conf
-
-    [service]
-
-    ...
-
-    server=demodb,testdb
-
-    ...
+| If the database name is not specified, the **status** command displays status information of the Java stored procedure server of every currently running database server.
```

Comment:
Is it correct to edit here? If it is correct, the korean manual should also be edited.

Reply:
Thank you. This fix is incorrect. I'm gonna revert it.